src/ngraph_op_kernels.cc

```diff
@@ -114,18 +114,22 @@ class NGraphIdentityOp : public OpKernel {
 REGISTER_KERNEL_BUILDER(Name("NoOp").Device(ngraph_bridge::DEVICE_NGRAPH),
                         NGraphNoOp);
 
-REGISTER_KERNEL_BUILDER(Name("Const")
-                            .Device(ngraph_bridge::DEVICE_NGRAPH)
-                            .TypeConstraint("dtype",
-                                            {DT_FLOAT, DT_INT32, DT_BOOL}),
-                        NGraphConstOp);
-
-#define REGISTER_IDENTITY(T)                                        \
-  REGISTER_KERNEL_BUILDER(Name("Identity")                          \
-                              .Device(ngraph_bridge::DEVICE_NGRAPH) \
-                              .TypeConstraint<T>("T"),              \
-                          NGraphIdentityOp);
-
-REGISTER_IDENTITY(float);
-REGISTER_IDENTITY(int32);
-REGISTER_IDENTITY(int64);
+// A bit of unfortuante silliness here: TypeConstraint takes a gtl::ArraySlice
+// for the allowed types, but NGraphDTypes is an std::set and there is
```

Comment:
We can use gtl::ArraySlice as the storage for NGraphDataTypes - no?

Reply:
I reckon we can, actually. Will do